solvebio/query.py

```diff
@@ -830,6 +834,12 @@ def join(self, query_b, key, key_b=None, prefix="b_", always_prefix=True):
         # If no key_b is provided, use the same key as for A
         key_b = key_b or key
 
+        # Limit cannot be larger than 1000 * 100
```

Comment:
is a join limit needed?

Reply:
I thought it could be useful, however, a user can always do slicing if he doesn't want to work with a total number of rows.
Removed.